### PR TITLE
Show 'Use SR Metdata' when 'Subtitles' are checked

### PR DESCRIPTION
**Pull request by h3llrais3r**
Fixes https://github.com/SickRage/SickRage/issues/2612

Only show the option 'Use SR MetaData' when 'Subtitles' are checked in edit show.

- [ ] PR is based on the DEVELOP branch
- [ ] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [ ] Read [contribution guide](https://github.com/SickRage/SickRage/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked

- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

File: gui/slick/js/core.js [hunk 1]
```diff
@@ -2759,6 +2759,18 @@ var SICKRAGE = {
...
+            });
+
+            $(".enabler").on('click', function() {
+                if ($(this).prop('checked')){
+                    $('#content_'+$(this).attr('id')).fadeIn("fast", "linear");
+                } else {
+                    $('#content_'+$(this).attr('id')).fadeOut("fast", "linear");
+                }
+            });
+
```
Comment (sharkykh) [inline gui/slick/js/core.js]: In my mind these two "functions" should be moved to [`common.init`](https://github.com/SickRage/SickRage/blob/26531ccd3917ace8345ada7159d72e2fa9dc760f/gui/slick/js/core.js#L87) if it's being used in [more than one section](https://github.com/SickRage/SickRage/blob/26531ccd3917ace8345ada7159d72e2fa9dc760f/gui/slick/js/core.js#L289-L299).

Comment (miigotu) [reply gui/slick/js/core.js]: Done in https://github.com/h3llrais3r/SickRage/pull/1
@h3llrais3r please merge that lol, I cant push to your pr

Comment (h3llrais3r) [reply gui/slick/js/core.js]: Done 😄

**Review by sharkykh — APPROVED** [review]

**Pull request merged**